modules/vba_documents/lib/vba_documents/object_store.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module VBADocuments
+  class ObjectStore
+    # Obtain the configured VBADocuments bucket
+    def bucket
```

Comment:
Is there a need for the `bucket` outside the instance?

Reply:
Probably not, I'll clean that up along with refactoring a couple other classes to use the ObjectStore class